<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KGuestInfoCheck</class>
 <widget class="QWidget" name="KGuestInfoCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>按房间号查</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QComboBox" name="m_room_list"/>
              </item>
              <item>
               <widget class="QPushButton" name="m_search_roomid">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>按姓名查</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="m_name_edit"/>
              </item>
              <item>
               <widget class="QPushButton" name="m_search_name">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>按身份证查</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="m_idnumber_edit"/>
            </item>
            <item>
             <widget class="QPushButton" name="m_search_btn">
              <property name="text">
               <string>查询</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="m_table_view"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
